tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -3400,7 +3400,7 @@ declare namespace ts {
             markAsDirty(): void;
             getScriptFileNames(): string[];
             getLanguageService(): never;
-            getModuleResolutionHostForAutoImportProvider(): never;
+            getHostForAutoImportProvider(): never;
```

Comment:
is `never` our jokey way of saying "this has to be in the public API for some reason, but don't call it"?

Reply:
Not exactly; this method is required to implement `Project`, but the implementation on this class throws an error because an AutoImportProviderProject is not supposed to host another AutoImportProviderProject. The type got inferred from the throw.

Reply:
oh, so it's the current way of saying `throws NoImplementedError`